src/PrestaShopBundle/Form/Admin/Sell/Product/Shortcut/PriceShortcutType.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Form\Admin\Sell\Product\Shortcut;
+
+use Currency;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\MoneyType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Translation\TranslatorInterface;
+use Symfony\Component\Validator\Constraints\NotBlank;
+use Symfony\Component\Validator\Constraints\Type;
+
+class PriceShortcutType extends ShortcutType
+{
+    /**
+     * @var array
+     */
+    private $taxRuleGroupChoices;
+
+    /**
+     * @var array
+     */
+    private $taxRuleGroupChoicesAttributes;
+
+    /**
+     * @var Currency
+     */
+    private $defaultCurrency;
+
+    /**
+     * @param TranslatorInterface $translator
+     * @param array $locales
+     * @param array $taxRuleGroupChoices
+     * @param array $taxRuleGroupChoicesAttributes
+     * @param Currency $defaultCurrency
+     */
+    public function __construct(
+        TranslatorInterface $translator,
+        array $locales,
+        array $taxRuleGroupChoices,
+        array $taxRuleGroupChoicesAttributes,
+        Currency $defaultCurrency
+    ) {
+        parent::__construct($translator, $locales);
+        $this->taxRuleGroupChoices = $taxRuleGroupChoices;
+        $this->taxRuleGroupChoicesAttributes = $taxRuleGroupChoicesAttributes;
+        $this->defaultCurrency = $defaultCurrency;
+    }
+
+    /**
+     * @param FormBuilderInterface $builder
+     * @param array $options
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('price_tax_excluded', MoneyType::class, [
+                'required' => false,
+                'label' => $this->trans('Retail price (tax excl.)', 'Admin.Catalog.Feature'),
```

Comment:
There is an existing string in the same domain: 'Price(tax excl.)' don't you think we could keep this one? Especially since the label of the section is 'Retail price'
![image](https://user-images.githubusercontent.com/70583503/110100634-e9770f00-7da2-11eb-8bf6-8463b45edba3.png)

Reply:
I used the existing wording that are in the legacy page right now
![Capture d’écran 2021-03-15 à 11 16 08](https://user-images.githubusercontent.com/13801017/111138169-e5f03e80-857f-11eb-83ac-c75901357535.png)

I believe they were changed recently because there was a confusion with Cost price as well, maybe @marionf has more inputs on this as, if I remember correctly, she provided the new wordings? (I might be wrong I'm not sure any more)

Reply:
Ok, I see, thanks for the clarification @jolelievre. I didn't know there was a confusion with the Cost price. Let's keep 'retail price' then!